### PR TITLE
Add weekly meeting for Embedded WG

### PR DESCRIPTION
**Pull request by therealprof**
Not quite sure whether it's okay to add ourselves to the all list since we don't sit under any other team but I'll let others be the judge of that. 😉

### CODE REVIEW

File: wg-embedded.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name = "Rust Embedded working group"
+description = "Meetings for the Rust Embedded Working Group"
+
+[[events]]
+uid = "1704827243369"
+title = "Rust Embedded WG meeting"
+description = "Weekly team meeting of the Clippy team"
```
Comment (traviscross) [inline wg-embedded.toml:7]:
```suggestion
description = "Weekly team meeting of the embedded working group."
```

**Comment by traviscross** [comment]
A link should also be added to `README.md`.

File: all.toml [hunk 2]
```diff
@@ -8,4 +8,5 @@ includes = [
   "infrastructure.toml",
   "language.toml",
   "library.toml",
+  "wg-embedded.toml",
```
Comment (traviscross) [inline all.toml:11]: A link should also be added to `README.md`.

**Comment by therealprof** [comment]
Thanks for the feedback, should be resolved now.

**Comment by jonathanpallant** [comment]
> since we don't sit under any other team 

It sits under the Launching Pad team, which you could use as a placeholder to sit under if you wish.

Also, it's technically called the "Embedded devices working group" (see https://github.com/rust-lang/team/blob/a8db5278f31f2f2ccbad7abceea27bbdbb16e02d/teams/wg-embedded.toml#L54). Probably makes sense to be consistent between here and rust-lang/team regardless of what the group actually calls itself.

File: wg-embedded.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+description = "Meetings for the Rust Embedded Working Group"
+
+[[events]]
+uid = "1704827243369"
+title = "Rust Embedded WG meeting"
+description = "Weekly team meeting of the embedded wg"
+location = "#rust-embedded on matrix.org (hhttps://app.element.io/#/room/#rust-embedded:matrix.org)"
+last_modified_on = "2024-01-09T20:00:00.00Z"
+start = { date = "2024-01-09T20:00:00.00", timezone = "Europe/Amsterdam" }
+start = { date = "2024-01-09T21:00:00.00", timezone = "Europe/Amsterdam" }
```
Comment (davidtwco) [inline wg-embedded.toml:11]:
```suggestion
end = { date = "2024-01-09T21:00:00.00", timezone = "Europe/Amsterdam" }
```

**Comment by therealprof** [comment]
> It sits under the Launching Pad team, which you could use as a placeholder to sit under if you wish.

Well, as far as I am concerned, this team could vanish just as quickly as it was introduced. Since it is not mentioned on the Rust governance site, I'd rather not go down the rabbit hole trying to introduce this here.

> Also, it's technically called the "Embedded devices working group"

Point taken, will adjust.

**Comment by therealprof** [comment]
Third time is a charm... maybe? 😅

**Review by davidtwco — APPROVED** [review]

**Pull request merged**